ddsp/training/train_util.py

```diff
@@ -25,12 +26,21 @@
 
 
 # ---------------------- Helper Functions --------------------------------------
-def get_strategy(tpu='', gpus=None):
+def get_strategy(tpu='', cluster_config=''):
   """Create a distribution strategy.
 
   Args:
     tpu: Address of the TPU. No TPU if left blank.
-    gpus: List of GPU addresses for synchronous training.
+    cluster_config: VM specific string for cluster configuration dict
```

Comment:
This is a good start for the docstring. Couple thoughts. 

* Please don't indent on wrapped docstrings as following the google style guide.
* Can you make it clear up front that this is only for multi-worker gpu jobs, and that for multi-gpu jobs on a single work, just call the function with no arguments?
* Can you also add a web link to more info about specifying a cluster? (i.e. it's not clear what `in the TF_CONFIG` format means)
* Also, I'm not sure you should specifically refer to VMs here, as people may have local clusters that aren't VMs. It's more just about multi-worker vs. single worker.

Reply:
- Done
- I added this information in the description section of the docstring as it references both non-obligatory arguments. Would it be okay?
- Done 
- Yes, you're right, I made a change

>